packages/plugins/rpc-manager/src/lib/index.ts

```diff
@@ -47,13 +102,15 @@ export default class RpcManager {
       EthSignData,
       EthSubscribe,
       EthUnsubscribe
-    ].map((rpcModifier) => new rpcModifier(this.embark, this.rpcModifierEvents));
+    ].map((rpcModifier) => new rpcModifier(this.embark, this.rpcModifierEvents, nodeAccounts, accounts, web3));
```

Comment:
It's purely a stylistic thing, so feel free to ignore, but it "looks weird" to me to use `new` with a function that has a name that starts with a lowercase letter. How about `.map((RpcMod) => new RpcMod(...))`?

Reply:
Done.